CondFormats/JetMETObjects/interface/METCorrectorParameters.h

```diff
@@ -58,13 +59,15 @@ class METCorrectorParameters
         float parameter(unsigned fIndex)    const {return mParameters[fIndex];        }
         std::vector<float> parameters()     const {return mParameters;                }
         unsigned nParameters()              const {return mParameters.size();         }
+	std::string MetAxis()  const {return mMetAxis;       }
         int operator< (const Record& other) const {return xMin(0) < other.xMin(0);    }
       private:
         //-------- Member variables ----------
         unsigned           mNvar;
         std::vector<float> mMin;
         std::vector<float> mMax;
         std::vector<float> mParameters;
+	std::string        mMetAxis;
```

Comment:
This data member addition is a schema change... To be clear, whenever a change affects directly or indirectly a top level class, you should create a new class.